README.rst

```diff
@@ -25,7 +25,10 @@ In order to build the Snowflake PHP PDO Driver, you must have the following soft
   - gcc 5.2 or higher
   - cmake
 
-- On macOS: clang
+- On macOS:
+
+  - clang
+  - cmake
```

Comment:
Good catch! Can you specify that we require cmake 2.8+

Reply:
Done -- thanks!